views/item/new.ejs

```diff
@@ -1,35 +1,99 @@
 <%- contentFor('title') %> EcoFoodie
 <%- contentFor('head') %>
+
+<link rel="stylesheet" type="text/css" href="/stylesheets/itemNew.css">
 <script type="application/javascript" src="/scripts/uploader.js"></script>
 <%- contentFor('body') %>
-<h1>New Items</h1>
+
+
+
+<img src="/images/greenArrow.png" alt="green arrow" style="width:128px;height:128px;">
+<img src="/images/EcoFoodieLOGO.png" alt="Eco Foodie Logo" style="width:128px;height:128px;">
+
 
 <form method="post" action="/item">
   <div id="other" class="form-group" style="display: none;">
     <input type="text" class="form-control" name="other" value="" placeholder="Please describe...">
   </div>
 
-  <div class="form-group">
+<!--  <div class="form-group">
     <label>Restaurant</label>
     <textarea class="form-control" rows="2" name="restaurantId" ></textarea>
-  </div>
+  </div> -->
 
   <div class="form-group">
-    <label>Picture</label>
+    <h3>Picture</h3>
     <input type="file" class="form-control" data-name="photo" />
   </div>
 
-  <div class="form-group">
-    <label>Bin</label>
-    <input type="textarea" class="form-control" name="bin" value="" />
+
+  <h2>What is this photo about?</h2>
+  <br>
+
+  <select id="category">
+  <option value="Plastic">Plastic</option>
+  <option value="LandFill">LandFill</option>
+  <option value="Recycle">Recycle</option>
+  </select>
+  <br>
+  <br>
+
+    <h1>Bin</h1>
+
+
+    <br>
+
+    <div class="radio-toolbar">
+        <input type="radio" id="radioApple" name="radioFruit" value="apple" checked>
+        <label for="radioApple">Recycling   <div class="left-img1"></div></label>
```

Comment:
Can you update the id and for attribute names so that they better describe what these form tags represent? It's no longer an Apple...

Also, the name attribute is currently radioFruit. There is no radioFruit attribute in the Item model in the EcoFoodie app. Instead, name should be the "bin" attribute (i.e. which "bin" does this item go into?).

The bin attribute is defined as an Integer value. Let's use the following values for the different bins:

Trash = 0
Recycling = 1
Compost = 2